docs/content/_navigation.json

```diff
@@ -242,19 +242,19 @@
         ]
       },
       {
-        "title": "Dagit UI & GraphQL API",
+        "title": "Dagster webserver & UI",
         "children": [
           {
-            "title": "Dagit UI",
-            "path": "/concepts/dagit/dagit"
+            "title": "Dagster UI",
+            "path": "/concepts/webserver/ui"
           },
           {
             "title": "GraphQL API",
-            "path": "/concepts/dagit/graphql"
+            "path": "/concepts/webserver/graphql"
           },
           {
             "title": "GraphQL Python client",
-            "path": "/concepts/dagit/graphql-client"
+            "path": "/concepts/webserver/graphql-client"
```

Comment:
we need to make sure we add redirects for these links: https://github.com/dagster-io/dagster/blob/master/docs/next/util/redirectUrls.json

caveat: this will be changed soon with the markdoc infra improvements, but for now, you will still need to hold off the redirects updates in a separate pr and land it after the docs are released -- this is because the docs are versioned (i.e. new pages will show up only after the release), but the redirects are part of the nextjs infra which is deployed continuously (i.e. redirects will apply to the current latest version).

Reply:
I can add that, but IIUC that is a separate PR to be merged after the release-- is there anything else you've identified in this PR, or are we good to merge?